macosx/Controller.mm

```diff
@@ -252,6 +252,7 @@ @interface Controller ()
 
 @property(nonatomic, readonly) NSMutableArray* fTorrents;
 @property(nonatomic, readonly) NSMutableArray* fDisplayedTorrents;
+@property(nonatomic, readonly) NSMutableDictionary* fTorrentHashes;
```

Comment:
I would not use Hungarian notation as we don't have property/ivar division anymore. I am actually planning to remove it entirely in a future PR.

Reply:
I have no idea what this means

Reply:
I looked it up....
I used the name just to follow conventions already used